### PR TITLE
DEV-4208: As a user, I need the ability to delete attendances for a student

### PR DESCRIPTION
**Pull request by nlc**

### CODE REVIEW

**Review by jw81 — APPROVED** [review]

File: lib/kicksite_svc_client/app/models/schools/activity.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ class Activity < Kicksite::NoSvcObject
       FAMILY_TYPE = 'family'.freeze
       STUDENT_TYPE = 'student'.freeze
       PROGRAM_TYPE = 'program'.freeze
+      ATTENDANCE_TYPE = 'attendance'.freeze
```
Comment (SlakrHakr) [inline lib/kicksite_svc_client/app/models/schools/activity.rb:14]: im confused... why is this needed here? these are for the recent activity section of the new dashboard. this ticket isn't adding attendance to that view is it?

Comment (SlakrHakr) [inline lib/kicksite_svc_client/app/models/schools/activity.rb:14]: im confused... why is this needed here? these are for the recent activity section of the new dashboard. this ticket isn't adding attendance to that view is it?

**Review by SlakrHakr — CHANGES_REQUESTED** [review]

File: lib/kicksite_svc_client/app/models/schools/activity.rb (hunk 1 above)
Comment (jw81) [reply lib/kicksite_svc_client/app/models/schools/activity.rb:14]: Ah I totally missed that it was in the activity model.
If we're just wanting to be able to reference a constant when creating a system-event, maybe storing that constant in something more generic like the [people model](https://github.com/KicksiteDev/kicksite_svc_client/blob/master/lib/kicksite_svc_client/app/models/schools/person.rb) would be better?

Comment (nlc) [reply lib/kicksite_svc_client/app/models/schools/activity.rb:14]: Right, I meant to remove this PR. The real change is in the events client.
